linchpin/provision/roles/dependencies/tasks/main.yml

```diff
@@ -10,11 +10,11 @@
   loop_control:
     loop_var: indv_prov
 
-- name: Install os-specific depdendencies:: Fedora
+- name: "Install os-specific depdendencies Fedora"
   include_tasks: "setup_dnf_deps.yml"
   when: setup_providers | length == 0  and ansible_distribution == 'Fedora'
 
-- name: Install os-specific depdendencies:: CentOS/RHEL
+- name: "Install os-specific depdendencies CentOS/RHEL"
```

Comment:
Are you sure about the quotes you added? 
Another thing you have typo there depdendencies -> dependencies

Reply:
Adding special characters like "::" causes failures in ansible. 
The is the reason I have added quotes to name. Else it is not necessary. 
I will fix typo

Reply:
Sure, I understand that.
But you removed "::" from the name, therefore you don't need quotes.

1. Remove "::" and you don't need quotes.
2. Keep "::" and add quotes.

And typo is not fixed yet.

Reply:
Why even have the double colon?